parcels/particleset.py

```diff
@@ -467,10 +470,10 @@ def nearest_index(array, value):
         if time_origin is 0:
             timestr = ' after ' + str(delta(seconds=show_time)) + ' hours'
         else:
-            timestr = ' on ' + str(time_origin + delta(seconds=show_time))
+            timestr = ' on ' + str(time_origin + pd.Timedelta(show_time, 's'))
```

Comment:
Can't we do this without `pandas`? If we need pandas only for the plotting of a figure title, then I think we really need to reconsider whether we're not better off not doing these figure titles